openstack/dms/v2/rabbitmq/instances/requests.go

```diff
@@ -93,6 +96,26 @@ type CreateOps struct {
 
 	// Indicates the tags of the instance
 	Tags []tags.ResourceTag `json:"tags,omitempty"`
+
+	// Indicates the parameter related to the yearly/monthly billing mode.
+	BssParam BssParam `json:"bss_param,omitempty"`
```

Comment:
```suggestion
	BssParam *BssParam `json:"bss_param,omitempty"`
```
the type should be a pointer